src/xml/EllipseXML.java

```diff
@@ -0,0 +1,41 @@
+package xml;
+
+import view.PencilState;
+import model.specific_path.Ellipse;
+
+public class EllipseXML extends ShapeXML {
+	
```

Comment:
What if we also create a ShapeXMLFactory that creates EllipseXML, DrawingXML, ...etc?